### PR TITLE
London Class 7-Ali-Nosratipour-Terminal-coursework-week-1

### PR DESCRIPTION
**Pull request by alinosratipour**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

**Comment by NeginDn** [comment]
Hi Ali,
 I didn't find any problem to comment on. 
 Well done!

File: MyDocuments/Quiz/QUIZ.md [hunk 1]
```diff
@@ -4,37 +4,37 @@ You can mark your choice by putting an `x` in the correct box
 
 **Which of these is incorrect?**
 
-- [ ] The terminal is an interface that allows you to access the command line
+- [x] The terminal is an interface that allows you to access the command line
 - [ ] It can be much faster to complete some tasks using a Terminal than with a GUI
 - [ ] You have access to many more commands and scripts on the Terminal than with a GUI
 - [ ] The terminal gives you more powers/privileges to complete tasks than a GUI
```
Comment (omarsharif-gr) [inline MyDocuments/Quiz/QUIZ.md:10]: I get the logic here, but a further look may be needed here? Take a look at the 'Now That You’re In, Let’s Use the Command Line' section of the following website: https://blog.galvanize.com/how-to-use-the-terminal-command-line/ .

File: MyDocuments/Quiz/QUIZ.md [hunk 2]
```diff
@@ -75,20 +75,20 @@ You can mark your choice by putting an `x` in the correct box
...
 - [ ] clear
 - [ ] mkdir
 
 **What is the correct name for the part after the - in this command "ls -all"?**
 
-- [ ] parameter
+- [x] parameter
 - [ ] extra
 - [ ] function
 - [ ] dash
```
Comment (omarsharif-gr) [inline MyDocuments/Quiz/QUIZ.md:94]: I believe that there should be a third option called 'argument' here as further outlined on this website: https://bash.cyberciti.biz/guide/Shell_command_line_parameters .
Do correct me if I am wrong.

**Review by omarsharif-gr — COMMENTED** [review]
Hi Ali, Great work! I have made some small comments that you may like to address. Have a great day.

**Pull request closed without merging**